be/src/runtime/data_stream_recvr.cc

```diff
@@ -307,8 +307,8 @@ Status DataStreamRecvr::SenderQueue::_build_chunk_meta(const ChunkPB& pb_chunk)
 
 Status DataStreamRecvr::SenderQueue::add_chunks(const PTransmitChunkParams& request, ::google::protobuf::Closure** done,
```

Comment:
Why the sender and receiver communicate through brpc if they are of the same process? Am I misunderstood？

According to your description, I think that sender and receiver can communicate through local channel (basically, communicating through memory)

Reply:
No, your understanding is correct. 

This PR is to optimize data flow and save the cost of chunk serilization/deserialization, and keep control flow(brpc) unchanged.